udata_front/theme/gouvfr/assets/js/api/reports.ts

```diff
@@ -0,0 +1,33 @@
+import { api, type User } from "@etalab/data.gouv.fr-components";
+import { AxiosResponse } from "axios";
+
+export type ReportReason = string;
+
+export type ReportReasons = Record<string, string>;
+
+export function getReportReasons() {
+  return api.get<ReportReasons>("/reports/reasons/")
+    .then(res => res.data);
+}
```

Comment:
Why don't you use `async`?
```suggestion
export async function getReportReasons() {
  const res = await api.get<ReportReasons>("/reports/reasons/")
  return res.data
}
```